test/dom-tests.js

```diff
@@ -100,12 +115,17 @@ describe.only('Dom Update Object', function() {
   })
 
   describe('listTags', function() {
-    it.skip('should return nothing', function() {
-
-    })
-
-    it.skip('should run insertAdjacentHTML for each element of allTags', function() {
+    it('should run insertAdjacentHTML for each element of allTags', function() {
+      let listSpy = makeSpy(domUpdates.listTags);
+      let allTags = [
+        'breakfast',
+        'lunch',
+        'dinner'
+      ]
 
+      listSpy(allTags, node);
+      expect(node.insertAdjacentHTML.calls).to.equal(3);
+      expect(node.insertAdjacentHTML.returned).to.deep.equal(['KJ', 'KJ', 'KJ']);
```

Comment:
Lol I love that you test everything is just you testing yourself. 'KJ' 'KJ' 'KJ' 'KJ'